blockstack/blockstackd.py

```diff
@@ -3782,8 +3780,12 @@ class Program {
             name = {}
             name['name'] = name_info['name']
             name['address'] = b58ToC32(str(name_info['address']))
+
             name['expire_block'] = name_info['expire_block']
-            name['registered_at'] = max(name_info['first_registered'], name_info['last_renewed'])
+            if name['expire_block'] != -1: # for special flag `-1` do not subtract the block height
```

Comment:
I'm not sure the Stacks chainstate boot code even uses `expire_block`.  Should we just remove it?  cc @lgalabru

Reply:
correct

Reply:
`expire_block` column removed! The PR description includes an updated sample of what the export data now looks like.